Specifications/model/information.model.md

```diff
@@ -0,0 +1,151 @@
+# Dataspace Information Model
+
+The information model defines the core concepts, entities, and relationships that underpin a `Dataspace`.
+
+## 1 Terminology
+
+### Dataspace
+
+A `Dataspace` is a set of technical services that facilitate interoperable asset sharing between entities.
```

Comment:
Can you propose an alternative formulation?

Reply:
Maybe something like:

"A `dataspace` is a defined set of entities that exchange data under contractually agreed terms of use. Data in this context does not necessarily mean a dataset or a file, but a determinable set of information (also in the form of an application or accesses to services or similar). From a business point of view, a `dataspace` is an association of a set of participants who want to exchange data offers. From a technical point of view, a `dataspace` is a set of technical services that facilitate interoperable asset sharing between entities.

Reply:
Maybe something like:

"A `dataspace` is build of a defined set of entities that exchange data under contractually agreed terms of use. Data in this context does not necessarily mean a dataset or a file, but a determinable set of information (also in the form of an application or accesses to services or similar). From a business point of view, a `dataspace` is an association of a set of participants who want to exchange data offers. From a technical point of view, a `dataspace` is a set of technical services that facilitate interoperable asset sharing between entities.

Reply:
IMO we should only focus on the technical definition in these documents because they are specifications. The other parts of your definition would fit well in the RAM documents and supporting material.